src/components/map/map.tsx

```diff
@@ -9,7 +10,14 @@ const Map: React.FC = () => {
 
   useMapSetup(mapContainer);
 
-  return <div ref={mapContainer} className="h-screen" />;
+  return (
+    <div className="grid grid-cols-1 grid-rows-1">
+      <div className="z-[1000] col-start-1 row-start-1 flex items-end justify-end">
```

Comment:
this div stretches across the entire map and blocks the user from interacting with the map (no zoom, moving around etc)